PHP/PHP Source.sublime-syntax

```diff
@@ -820,7 +820,7 @@ contexts:
                   scope: punctuation.section.group.begin.php
                   set:
                     - meta_scope: meta.function.parameters.php meta.group.php
-                    - match: '{{visibility_modifier}}'
+                    - match: '(?i){{visibility_modifier}}|\b(?:readonly)\b'
```

Comment:
I'm curious about the case insensitivity.

Reply:
I don't mind either way, I went with what was used in [the one other place that `readonly` shows up](https://github.com/sublimehq/Packages/blob/065a2b7b27066908b202e4d0ffcd45acf097a9cc/PHP/PHP%20Source.sublime-syntax#L258-L260) and [the part for typehints](https://github.com/sublimehq/Packages/blob/065a2b7b27066908b202e4d0ffcd45acf097a9cc/PHP/PHP%20Source.sublime-syntax#L400-L405). Feels to me like the same idea as `public`/`protected`/`private` or `int`/`string`/`array`/etc. in function arguments, which as far as I know all technically _can_ be uppercase.

Reply:
Okay. If I were doing this, I'd change it to `'{{visibility_modifier}}|\b(?i:readonly)\b'`, but otherwise sounds good.

Reply:
If all modifiers may be upper-case the proposed solution is correct as `{{visibility_modifier}}` matches case-sensitive only. I'd probably just skip the group, but that's not a big deal.

```suggestion
                    - match: '(?i){{visibility_modifier}}|\breadonly\b'
```